packages/step-vehicle-inspection/LICENSE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
Should we use MIT for everything? I thought we were on apache2

Reply:
We should use apache2. None of the modules will need license file. Going to remove it.
License is in package.json

Reply:
Btw. BSD license for demo apps makes sense.